app/src/main/java/com/duckduckgo/app/browser/webview/WebViewActivity.kt

```diff
@@ -40,7 +40,7 @@ class WebViewActivity : DuckDuckGoActivity() {
     @Inject
     lateinit var webViewClient: BrowserWebViewClient
 
-    private val binding: ActivityEmailWebviewBinding by viewBinding()
+    private val binding: ActivityWebviewBinding by viewBinding()
```

Comment:
This activity is used in different places but was using the email webview layout so I'm renaming that layout to something that makes more sense